packages/components/src/ThumbnailGallery/ThumbnailGallery.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from "react";
+import styles from "./ThumbnailGallery.css";
+import { FormatFile } from "../FormatFile";
+import { LightBox } from "../LightBox";
+
+interface Attachment {
```

Comment:
We have a `FileUpload` interface in `packages/components/src/InputFile/InputFile.tsx` that's probably good to use here.

Reply:
I was initially using that but it looks like FileUpload has the progress property which in this case would always be 1. So thought of having another interface specific for Gallery that does not require progress key but would be passing it as 1 when it is sending it to FormatFile.